packages/backend-next/src/index.ts

```diff
@@ -39,7 +39,7 @@ backend.add(
   import('@backstage/plugin-permission-backend-module-allow-all-policy'),
 );
 backend.add(import('@backstage/plugin-permission-backend/alpha'));
-backend.add(import('@backstage/plugin-proxy-backend'));
+backend.add(import('@backstage/plugin-proxy-backend/alpha'));
```

Comment:
worth calling out in a changeset that end users need to do this too, for the relevant plugins? Technically it's even a breaking change for those APIs (removing things from the public surface), but since it's backend system of course we'll see is as accidental and not bother :)

Reply:
as discussed, end users already need to use `/alpha`, it's just in the main repo where we didn't